src/seedu/addressbook/AddressBook.java

```diff
@@ -579,6 +588,47 @@ private static String executeListAllPersonsInAddressBook() {
         return getMessageForPersonsDisplayedSummary(toBeDisplayed);
     }
 
+    /**
+     * Displays all persons in the address book to the user; sorted by second command word
+     * @return feedback display message for the operation result
+     */
+    private static String executeSortAllPersonsInAddressBook(String arg) {
```

Comment:
Normally function name can be shorten to highlight the key important points e.g. `executeSortAddressBook` or `executeSortAllPersons` both sort of implies the other.